src/app/bookmarks/pages/bookmarks-search/bookmarks-search.component.scss

```diff
@@ -1,7 +1,8 @@
-.image-object {
-  object-fit: contain;
-  vertical-align: middle;
-}
+@import '../../../_bm_mixins.scss';
```

Comment:
@HenryT-CG I tried to use the mixins with absolute paths like you did in other apps, but for some reason it couldn't resolve the path during compilation. Any recommendations?

Reply:
will check this in next step